BILL_OF_MATERIALS.md

```diff
@@ -0,0 +1,43 @@
+# Bill Of Materials (BOM)
+
+
+## Alexander Koch arms
+
+### Follower and Leader arms
+
+TODO
+
+### Leader arm only
+
+| Part | Amount | Cost/p US | Buy US | Cost/p EU | Buy EU | Cost/p UK | Buy UK |
```

Comment:
Nit: It seems like the Buy UK column isn't complete. I think if we have this column we should try and fill it out more.